flow/layers/container_layer.cc

```diff
@@ -184,6 +186,11 @@ void ContainerLayer::PaintChildren(PaintContext& context) const {
   // layer calls PaintChildren(), though, it may have modified the
   // PaintContext so the test doesn't work in this "context".
 
+  // Apply any outstanding state that the children cannot individually
+  // and collectively handle.
+  auto restore = context.state_stack.applyState(
```

Comment:
Here, There may be a redundant saveLayer operation here。
```
OpacityLayer                 
     |_ColorFilterLayer
                 |_DL
```
it will be :
```
// ContainerLayer applyState
{
   setAlpha(..)
   saveLayer()
   // CF paint
   { 
      setCF(...)
      saveLayer(....)
   }
}

```